// deckboard-kit/dist/constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ICONS = exports.PLATFORMS = exports.INPUT_METHOD = void 0;
var INPUT_METHOD;
(function (INPUT_METHOD) {
    INPUT_METHOD["INPUT_TEXT"] = "input:text";
    INPUT_METHOD["INPUT_KEY"] = "input:key";
    INPUT_METHOD["INPUT_SELECT"] = "input:select";
    INPUT_METHOD["INPUT_FILE"] = "input:file";
    INPUT_METHOD["INPUT_FOLDER"] = "input:folder";
    INPUT_METHOD["INPUT_COLOR"] = "input:color";
    INPUT_METHOD["INPUT_CHECKBOX"] = "input:checkbox";
    INPUT_METHOD["INPUT_TEXTAREA"] = "input:multilinetext";
})(INPUT_METHOD = exports.INPUT_METHOD || (exports.INPUT_METHOD = {}));
;
var PLATFORMS;
(function (PLATFORMS) {
    PLATFORMS["WINDOWS"] = "WINDOWS";
    PLATFORMS["MAC"] = "MAC";
    PLATFORMS["LINUX"] = "LINUX";
    PLATFORMS["SUN"] = "SUN";
    PLATFORMS["OPENBSD"] = "OPENBSD";
    PLATFORMS["ANDROID"] = "ANDROID";
    PLATFORMS["AIX"] = "AIX";
})(PLATFORMS = exports.PLATFORMS || (exports.PLATFORMS = {}));
;
var ICONS;
(function (ICONS) {
    ICONS["FONTAWESOME_SOLID"] = "fas";
    ICONS["FONTAWESOME_BRAND"] = "fab";
    ICONS["IONICONS"] = "ion";
})(ICONS = exports.ICONS || (exports.ICONS = {}));
